e2e/terraform/README.md

```diff
@@ -147,26 +150,55 @@ Common commands:
   - Fix these errors and re-verify with `terraform plan` before running the `terraform apply` command.
 - Run the command `terraform apply` to initiate creation of the infrastructure.
```

Comment:
to initiate infrastructure recreation

Reply:
recreation or creation? we do not have the infrastructure yet, at this point of deployment.

Reply:
creation...my bad.